services/src/main/java/org/keycloak/services/error/KeycloakErrorHandler.java

```diff
@@ -150,8 +150,20 @@ private RealmModel resolveRealm(KeycloakSession session) {
         attributes.put("url", new UrlBean(realm, theme, session.getContext().getUri().getBaseUri(), null));
         attributes.put("locale", new LocaleBean(realm, locale, session.getContext().getUri().getBaseUriBuilder(), messagesBundle));
 
+        String errorKey;
```

Comment:
This change is not essential for this commit. But without it, the `ForbiddenException` throws from `Auth#require(string)` end up being presented as an internal error (which is not great for the user experience).